e2e/tests/001.sh

```diff
@@ -26,7 +26,7 @@ source "${LIBDIR}/k8s.sh"
 
 kubeconfig="$HOME/.kube/config"
 
-workload_cluster_pkg_rev=$(kpt alpha rpkg get --name nephio-workload-cluster --revision v7 -o jsonpath='{.metadata.name}')
+workload_cluster_pkg_rev=$(kpt alpha rpkg get --name nephio-workload-cluster --revision v8 -o jsonpath='{.metadata.name}')
```

Comment:
You may already be thinking about pulling this value for version automatically instead of this being hard coded, right?